service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -46,77 +47,83 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*cor
 		}
 	}
 
-	var clusterCIDR string
-	var subnetID string
-	var networkID string
+	values := map[string]interface{}{
+		"baseDomain": key.BaseDomain(&cr, r.baseDomain),
+		"chartOperator": map[string]interface{}{
+			"cni": map[string]interface{}{
+				"install": true,
+			},
+		},
+		"cluster": map[string]interface{}{
+			"calico": map[string]interface{}{
+				"CIDR": podCIDR,
+			},
+			"kubernetes": map[string]interface{}{
+				"API": map[string]interface{}{
+					"clusterIPRange": r.clusterIPRange,
+				},
+				"DNS": map[string]interface{}{
+					"IP": r.dnsIP,
+				},
+			},
+		},
+		"clusterDNSIP": r.dnsIP,
+		"clusterID":    key.ClusterID(&cr),
+		"clusterCIDR":  "",
+		"provider":     "unknown",
+	}
```

Comment:
For my understanding: where are those values linked?

Reply:
Does it mean that we have a "shared" values file?

Reply:
Yes, for cluster-specific values.